Source/AFResult.swift

```diff
@@ -25,83 +25,3 @@
 /// `Result` that always has an `Error` `Failure` type.
 public typealias AFResult<T> = Result<T, Error>
```

Comment:
There's really no reason to have just this `typealias` in its own file. It can be combined with the `Result` extensions.

Reply:
I've moved the typealias to `Result+Alamofire.swift`